lte/gateway/c/oai/tasks/grpc_service/HaServiceImpl.h

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <grpc++/grpc++.h>
+#include <grpcpp/impl/codegen/status.h>
+
+#include "lte/protos/ha_service.grpc.pb.h"
+
+extern "C" {
+#include "log.h"
+#include "hashtable.h"
+}
+
+bool process_ue_context(
+    const hash_key_t keyP, void* const elementP, void* parameterP,
+    void** resultP);
+
+namespace magma {
+using namespace lte;
```

Comment:
Keep the namespace convention consistent
```suggestion
namespace lte {
```